docs/BuildSource.md

```diff
@@ -70,3 +70,8 @@ cmake --build ./build --config Release --target install
 # Alternatively, build in Visual Studio via its built-in support for
 # CMake or by opening the generated build/timescaledb.sln solution file.
 ```
+
+#### Older PostgreSQL versions
+TimescaleDB can be installed with PostgreSQL 9.6.6+ and 10.9+ versions. Supprot for these 
+versions of PostgreSQL is deprecated and no future releases are planned for them. We highly 
```

Comment:
```suggestion
versions of PostgreSQL is deprecated and no future releases are planned for them. We 
```

Reply:
I usually try to avoid adding intensifiers in text, but you can ignore this.

Reply:
Thank you for suggestion. Removed.